cloudformation/stacks/AutoSpotting/parameters.yaml

```diff
@@ -4,9 +4,9 @@
 
 # This file is handled automatically using the clouds-aws tool. It can be
 # installed using pip, or from https://github.com/elias5000/clouds-aws
-LambdaZipPath: nightly/lambda_build_458.zip
+LambdaZipPath: nightly/lambda_build_584.zip
 LambdaHandlerFunction: handler.Handle
 LambdaS3Bucket: cloudprowess
-LambdaMemorySize: '256'
+LambdaMemorySize: '1024'
```

Comment:
You might as well change it in [terraform](https://github.com/cristim/autospotting/blob/master/terraform/variables.tf#L45) as well ;)

Reply:
Done in  bcb5811